apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/monetary/Asset.java

```diff
@@ -95,18 +108,30 @@ private Asset(ResultSet rs, DbKey dbKey) throws SQLException {
         this.decimals = rs.getByte("decimals");
     }
 
+    /**
+     * @deprecated
+     */
     public static DbIterator<Asset> getAllAssets(int from, int to) {
```

Comment:
does @deprecated mean that the whole api will be replaced soon?

Reply:
Now that means 'do not use it anymore'.
Later after removing 'old rest api' we can removed those methods completely.